numpy/core/src/umath/ufunc_object.c

```diff
@@ -3036,6 +3036,8 @@ PyUFunc_Reduce(PyUFuncObject *ufunc, PyArrayObject *arr, PyArrayObject *out,
     const char *ufunc_name = ufunc_get_name_cstr(ufunc);
     /* These parameters come from a TLS global */
     int buffersize = 0, errormask = 0;
+    static PyObject *NoValue = NULL;
+    npy_cache_import("numpy", "_NoValue", &NoValue);
```

Comment:
Ah, this should actually be `if (... < 0) { return NULL; }` (wrapped as normal), in case the import fails

Reply:
`npy_cache_import` has a return type of `void`, so we can't do that.

Reply:
Ah, in that case, I mean you should check that `NoValue != NULL`